packages/components/src/navigable-container/test/tababble-container.tsx

```diff
@@ -32,11 +33,14 @@ const getTabbableContainerTabbables = () => [
 
 const originalGetClientRects = window.HTMLElement.prototype.getClientRects;
 
+const stubEventToOffset = ( _event: KeyboardEvent ) => undefined;
+
 describe( 'TabbableContainer', () => {
 	beforeAll( () => {
 		// Mocking `getClientRects()` is necessary to pass a check performed by
 		// the `focus.tabbable.find()` and by the `focus.focusable.find()` functions
 		// from the `@wordpress/dom` package.
+		// @ts-expect-error TODO: Don't know how to resolve
```

Comment:
Feel free to ignore, like done in https://github.com/WordPress/gutenberg/blob/0cb1c2e525671103719b11c495c50e732d0904a5/packages/components/src/tab-panel/test/index.tsx#L54

Reply:
That's a good comment.

Committed verbatim in https://github.com/WordPress/gutenberg/pull/49377/commits/049f3dccb44d7ac24b2b67d3b4f48241727e6a25